packages/xo-server-sdn-controller/src/index.js

```diff
@@ -627,6 +631,12 @@ class SDNController extends EventEmitter {
       return
     }
 
+    const xapi = host.$xapi
```

Comment:
Inline.

Reply:
`xapi` is used 3 times in the method so I'd rather not.